qa/suites/rbd/basic/base/install.yaml

```diff
@@ -1,3 +1,6 @@
 tasks:
 - install:
-- ceph:
+- cephadm:
+overrides:
+  ceph:
```

Comment:
Nit: so should these be `cephadm` task overrides or does the `cephadm` task pull the config out of `ceph`?

Reply:
https://github.com/ceph/ceph/blob/b6a2dfc777d5639965c2888b59958af224259e8e/qa/tasks/cephadm.py#L1030-L1050

and 

https://github.com/ceph/ceph/blob/cab87f956b06551db1eec88cd76ea34e235b6a71/qa/tasks/ceph.py#L379-L403

are more or less identical, but we probably should call the cephadm.py's version of it.